modulefiles/ufs_acorn.intel

```diff
@@ -0,0 +1,26 @@
+#%Module
+
+proc ModulesHelp {} {
+  puts stderr "\tcit - loads modules required for building and running UFS Model on Acorn"
+}
+
+module-whatis "loads UFS Model prerequisites for Acorn"
+
+module load PrgEnv-intel/8.1.0
+module load intel/19.1.3.304
```

Comment:
Can we have intel /2022 version?

Reply:
I don't know. At this moment this is the only compiler available.